ragna/ui/central_view.py

```diff
@@ -0,0 +1,681 @@
+
+        chat_documents_pills = []
+        if (
+            self.current_chat is not None
+            and "metadata" in self.current_chat
+            and "documents" in self.current_chat["metadata"]
+        ):
+            doc_names = [d["name"] for d in self.current_chat["metadata"]["documents"]]
+
+            for doc_name in doc_names:
+                pill = pn.pane.HTML(
+                    f"""<div class="chat_document_pill">{doc_name}</div>""",
+                    stylesheets=[
+                        """
+                                                 :host {
+                                                    background-color: rgb(241,241,241);
+                                                    margin-top: 15px;
+                                                    margin-left: 5px;   
+                                                    margin-right: 5px;
+                                                    padding: 5px 15px;
+                                                    border-radius: 10px;
+                                                    color:var(--accent-color);
+                                                    
+                                                 }   
+
+                                                 """
+                    ],
+                )
+
+                chat_documents_pills.append(pill)
+
+        chat_info_button = None
+        if self.current_chat is not None:
+            chat_info_button = pn.widgets.Button(
+                name="Chat Info", button_style="outline", icon="info-circle"
+            )
+            chat_info_button.stylesheets.append(
+                """
+                                        :host {  
+                                                    margin-top:10px;
+                                        }
+                                        
+                                    """
+            )
+            chat_info_button.on_click(self.on_click_chat_info_wrapper)
+
+        return pn.Row(
+            chat_name_header,
+            *chat_documents_pills,
+            chat_info_button,
+            stylesheets=[
+                """:host {  
+                                    background-color: #F9F9F9;
+                                    border-bottom: 1px solid #EEEEEE;
+                                    width: 100% !important;
+                                    margin:0px;
+                                    height:54px;
+                                }
+
+                                :host div {
+                                    
+                                    vertical-align: middle;
+                                }
+"""
+            ],
+        )
+
+    def set_loading(self, is_loading):
+        self.main_column.loading = is_loading
+
+    def __panel__(self):
+        """
+        The ChatInterface.view_latest option doesn't seem to work.
+        So to scroll to the latest message, we use some JS trick.
```

Comment:
We need to report this to `panel`. @pierrotsmnrd Could you compile a list of stuff like this that we encountered so they can be fixed upstream. Not blocking for this PR.